website/settings/celeryconfig.py

```diff
@@ -0,0 +1,34 @@
+from celery.schedules import crontab
+
+
+##### Celery #####
+## Default RabbitMQ broker
+BROKER_URL = 'amqp://'
+
+# Default RabbitMQ backend
+CELERY_RESULT_BACKEND = 'amqp://'
+
+# CELERYBEAT_SCHEDULE_FILENAME = 'celeryconfig.py'
+# Setting up a scheduler
+CELERYBEAT_SCHEDULE = {
+    '5-minute-emails': {
+        'task': 'notify.send_users_email',
+        'schedule': crontab(minute='*/5'),
```

Comment:
K. Thanks for the clarification. Not sure about linking proposals here. 

That's a question for @lbanner @GaryKriebel @erinmayhood . I'd personally find it useful for code review purposes to have the links in the PR descriptions, granted the links have the proper permissions set.